ui/src/features/components/Report/compareReports.js

```diff
@@ -72,6 +83,47 @@ class CompareReports extends React.Component {
         this.setMergedReports(reportsList);
     };
 
+    onEditSymbol = (value, index) => {
+        const {reportsList} = this.state;
+        const {newAggregateReports} = this.state;
+        const currentReport = newAggregateReports[index];
+        const originalAlias = currentReport.alias;
+
+        if (value === originalAlias) {
+            return;
+        }
+
+        currentReport.alias = value;
+        reportsList[index].name = value;
+        this.setState({reportsList: [...reportsList]});
+
+        for (let item in currentReport) {
+            if (item === 'errorsBarKeys' || item === 'errorsCodeGraphKeys' || item === 'latencyGraphKeys' || item === 'rpsKeys') {
+                currentReport[item].forEach((key, index, arr) => {
+                    arr[index] = arr[index].replace(originalAlias, value);
+                });
+            } else if (item === 'errorsBar' || item === 'errorsCodeGraph' || item === 'latencyGraph' || item === 'rps') {
+                for (let i = 0; i < currentReport[item].length; i++) {
+                    for (let key in currentReport[item][i]) {
+                        if (key.indexOf('_') !== -1) {
+                            let data = key.split('_')[1];
+                            currentReport[item][i][`${value}_${data}`] = currentReport[item][i][key];
+                            delete currentReport[item][i][key];
+                        }
+                    }
+                }
+            } else if (item === 'scenarios') {
+                let newName = currentReport[item][0]['name'];
+                newName = newName.split('_');
+                newName[0] = value;
+                newName = newName.join('_');
+                currentReport[item][0]['name'] = newName;
+            }
+        }
+
+        this.setMergedReports(reportsList, this.props.benchmark, true, newAggregateReports);
```

Comment:
change to 
        this.setMergedReports(reportsList, this.props.benchmark);

because its already ready from this.props.aggregateReports - which you change